lib/http/2/compressor.rb

```diff
@@ -134,9 +130,9 @@ def dup
       # @return [Array] +[key, value]+
       def dereference(index)
         # NOTE: index is zero-based in this module.
-        STATIC_TABLE[index] or
-          @table[index - STATIC_TABLE.size] or
-          raise CompressionError.new("Index too large")
+        STATIC_TABLE[index] ||
```

Comment:
The only difference between `or` and `||` is operator precedence - since there are no other operators involved in this expression, this change should be safe.

As a side note, `or`'s precedence is surprisingly low, which is why RuboCop suggests to avoid it.